plugins/standard/pages/conferences.js

```diff
@@ -0,0 +1,65 @@
+function joinConference(params) {
+    params = $.extend(params, { 'Digits': 1 });
+    window.parent.Client.call(params)
+}
+
+$(document).ready(function(){
+	jQuery(function($) {
+		$(function(){
```

Comment:
Heh, just noticed this, but `$(document).ready(function() {});`, `jQuery(function($) {});` and `$(function() {});` all do the same thing... so we've got 3 levels of code all waiting for DOMReady.